pkg/auth/rbac.go

```diff
@@ -201,6 +201,9 @@ func ReadRbacPolicy(dexEnabled bool) (policy map[string]*Permission, err error)
 
 // Checks user permissions on the RBAC policy.
 func CheckUserPermissions(rbacConfig RBACConfig, user *User, env, envGroup, application, action string) error {
+	if isEnvironmentIndependent(action) {
+		env = "*"
+	}
```

Comment:
please add test for this function specifically (rbac_test.go).
A test should for example ensure that `CheckUserPermissions(app1...` behaves the same as `CheckUserPermissions("*",...` (assuming permission is given to the app)

Reply:
Same here. This change was handled on a different PR.